bulk-ingester/src/test/java/org/alfresco/hxi_connector/bulk_ingester/processor/mapper/AlfrescoPropertyMapperTest.java

```diff
@@ -0,0 +1,489 @@
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.hxi_connector.bulk_ingester.processor.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.BOOLEAN;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.CONTENT_DATA_ID;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.DATE;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.DOUBLE;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.FLOAT;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.LONG;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.NODEREF;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.NULL;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.PATH;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.SERIALIZABLE;
+import static org.alfresco.elasticsearch.db.connector.model.PropertyValueType.STRING;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.util.SerializationUtils;
+
+import org.alfresco.elasticsearch.db.connector.model.AlfrescoNode;
+import org.alfresco.elasticsearch.db.connector.model.ContentMetadata;
+import org.alfresco.elasticsearch.db.connector.model.NodeProperty;
+import org.alfresco.elasticsearch.db.connector.model.PropertyKey;
+import org.alfresco.elasticsearch.db.connector.model.PropertyValue;
+import org.alfresco.elasticsearch.db.connector.model.PropertyValueType;
+
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.LinguisticNaming"})
+class AlfrescoPropertyMapperTest
+{
+    @Test
+    void shouldMapJustPropertiesWithGivenName()
+    {
+        // given
+        AlfrescoNode alfrescoNode = new AlfrescoNode();
+
+        String description = "description";
```

Comment:
I think this would be slightly clearer if the variable was called `propertyName`.

Reply:
fixed!